### ISSUE TITLE
Canada English PDFs

### ISSUE THREAD

**Issue by junglernr** [Muetze42/hellofresh-database#40]
**Describe the bug**
The Canadian English pdf links are leading to the French pdfs

**Comment by Muetze42** [comment]
The PDF links seem to be identical?

For example, when I open the PDF for this recipe, it doesn't matter whether I have set it to English or French at the top.
I always get an English PDF:

But I will compare this with the API. There must be a difference somewhere.

**Comment by Muetze42** [comment]
And I only ever get a French PDF via the API, regardless of whether I use “en-CA” or “fr-CA”. Just like on the website:
https://www.hellofresh.ca/recipes/seared-provencal-steak-573b64dbf8b25e2e7b8b4567

I don't know how it works in Canada. Do you always receive the PDF in a selected language from HelloFresh?

**Comment by Muetze42** [comment]
Ahh.
https://www.hellofresh.ca/recipes/broiled-salmon-and-garlicky-shrimp-66635bc845478430853ef638

Here exsting 2 PDFs:

* 66635bc845478430853ef638-**en-CA**.pdf
* 66635bc845478430853ef638-**fr-CA**.pdf

Unfortunately, some recipes only seem to be available in one language and some in both languages.

I will include the change, but for many recipes a PDF only exists in one language.
Presumably a recipe will only be translated as soon as it ends up in one of the cooking boxes.

**Comment by junglernr** [comment]
Yeah I've always gotten the recipe cards in English. When I use your links
to "view on hellofresh" and then click their link to download pdf I get the
English version.

**Comment by Muetze42** [comment]
Some recipes have both languages as PDF and others only 1 language. But I have no influence on that either.

With the new version, however, both languages are synchronized according to the language, even if 2 languages are available as PDF.

**Comment by Muetze42** [comment]
Fixed in v3.0.0! Canada now correctly supports both English and French locales.

  Note: PDF availability in both languages depends on what HelloFresh provides in their API.
